### ISSUE TITLE
Export some CSS variables

### ISSUE THREAD

**Issue by zgoda** [milligram/milligram#307]
It would be nice if Milligram exported at least colour variables so the most basic customisation would not require running through sass pipeline.

**Comment by fjeddy** [comment]
Agreed, you only support the latest browsers anyways, so why wouldn't you make multi-themes (Dark mode for instance) super easy and use CSS variables instead of SASS variables?

**Comment by Ayoub-glitsh** [comment]
Hi, I’d like to work on this issue and propose a solution using CSS variables.
Let me know if I can proceed with a pull request.
